src/pallets/static/style.css

```diff
@@ -8,27 +13,105 @@ header nav a.logo {
 }
 
 header nav a.logo img {
-  height: 3em;
-  max-width: none;
+  width: 3em;
+}
+
+header nav {
+  display: flex;
+  justify-content: space-between;
+  justify-items: center;
+  padding: 0;
+  margin: 0;
+}
+
+header nav ul {
+  padding: 0 1rem;
 }
 
 @media (max-width: 767px) {
   header nav {
-    overflow-x: auto;
+    overflow-x: hidden;
```

Comment:
I think `scroll` would be better, that way it's still possible to get to on very small screens.

Reply:
You'd be better changing the overall shape of the menu from row to column, and maybe centring the text when on mobile view. I don't think I have ever seen a mobile menu with a horizontal scroll bar